Source/JavaScriptCore/b3/B3LowerToAir.cpp

```diff
@@ -4469,6 +4469,25 @@ class LowerToAir {
         }
 
         case Select: {
+            if (m_value->type().isVector()) {
+                // Conditional moves aren't widely supported for vectors, so
+                // we lower vector Select to a branching construct.
+
+                auto ifTrueBlock = newBlock();
+                Air::BasicBlock* beginBlock;
+                Air::BasicBlock* doneBlock;
+                splitBlock(beginBlock, doneBlock);
+
+                append(Air::MoveVector, tmp(m_value->child(2)), tmp(m_value));
+                m_insts.last().append(createBranch(m_value->child(0)));
+                beginBlock->setSuccessors(ifTrueBlock, doneBlock);
+
+                ifTrueBlock->append(Air::MoveVector, m_value, tmp(m_value->child(1)), tmp(m_value));
+                ifTrueBlock->append(Air::Jump, m_value);
+                ifTrueBlock->setSuccessors(doneBlock);
+                return;
+            }
+
```

Comment:
This is the actual diff in `B3LowerToAir.cpp`, everything else is just trimmed trailing whitespace.

Reply:
Amended commit to remove all the whitespace changes.

Reply:
nit: Can we use `append()` instead of  `m_insts.last().append()` for `m_insts.last().append(createBranch(m_value->child(0)));`? Maybe we need a refactoring patch to replace all `m_insts.last().append()` with `append()`.